tools/fuchsia/merge_and_upload_debug_symbols.py

```diff
@@ -60,8 +60,19 @@ def ProcessCIPDPackage(upload, cipd_yaml, engine_version, out_dir, target_arch):
                      'fuchsia-debug-symbols-%s.cipd' % target_arch)
     ]
 
-  subprocess.check_call(command, cwd=_packaging_dir)
-
+  # Retry up to three times.  We've seen CIPD fail on verification in some
+  # instances. Normally verification takes slightly more than 1 minute when
+  # it succeeds.
+  num_tries = 3
+  while tries in range(num_tries):
+    try:
+      subprocess.check_call(command, cwd=_packaging_dir)
+      success = True
+      break
+    except subprocess.CalledProcessError:
+      print('Failed %s times' % tries + 1)
+      if tries == num_tries - 1:
+        raise
```

Comment:
Maybe raise the actual exception?

Reply:
Got rid of the success part - that was an experiment I forgot to get rid of.

`raise` should just reraise AFAIK.

Reply:
TIL